src/common/model_reader.py

```diff
@@ -56,15 +56,15 @@
 valid_relationship_types = ["many_to_one", "one_to_one", "many_to_many"]
 
 
-def get_list_values(list_str):
-    return [item.strip() for item in list_str.split(LIST_DELIMITER) if item.strip()]
+def get_list_values(list_str, list_delimiter):
```

Comment:
It seems this function is not used anywhere. If so, we can remove it.

Reply:
Removed